app/controllers/students_controller.rb

```diff
@@ -30,6 +30,18 @@ def edit
     authorize @student
   end
 
+  def find_missing
```

Comment:
This belongs in the model

Reply:
Also recommend you use [`CSV`](https://ruby-doc.org/stdlib-2.6.1/libdoc/csv/rdoc/CSV.html) instead of rolling your own

Reply:
The controller action itself should leverage `respond_to` format